src/routes/plans/[id]/+page.svelte

```diff
@@ -436,6 +469,7 @@
       >
         <CalendarIcon />
       </PlanNavButton>
+      <ExtensionMenu extensions={data.extensions} user={data.user} on:callExtension={onCallExtension} />
```

Comment:
Can you also add a title prop and then do something like `title={!compactNavMode ? 'Extensions' : ''}` so that this menu item is responsive like the ones to the left (ViewMenu is special i think)